src/spdx_tools/spdx3/model/ai/ai_package.py

```diff
@@ -22,17 +22,17 @@ class SafetyRiskAssessmentType(Enum):
 @dataclass_with_properties
 class AIPackage(Package):
     energy_consumption: Optional[str] = None
-    standards_compliance: List[str] = field(default_factory=list)
-    limitations: Optional[str] = None
+    standard_compliance: List[str] = field(default_factory=list)
+    limitation: Optional[str] = None
     type_of_model: List[str] = field(default_factory=list)
     information_about_training: Optional[str] = None
     information_about_application: Optional[str] = None
-    hyperparameters: Dict[str, Optional[str]] = field(default_factory=dict)
-    data_preprocessing_steps: Optional[str] = None
-    model_explainability_mechanisms: Optional[str] = None
+    hyperparameter: Dict[str, Optional[str]] = field(default_factory=dict)
+    model_data_preprocessing: Optional[str] = None
+    model_explainability: Optional[str] = None
     sensitive_personal_information: Optional[bool] = None
-    metrics_decision_thresholds: Dict[str, Optional[str]] = field(default_factory=dict)
-    metrics: Dict[str, Optional[str]] = field(default_factory=dict)
+    metrics_decision_threshold: Dict[str, Optional[str]] = field(default_factory=dict)
```

Comment:
```suggestion
    metric_decision_threshold: Dict[str, Optional[str]] = field(default_factory=dict)
```
Also for other occurrences.

Reply:
Good catch, thanks! I updated this, rebased and as I needed to force-push anyway, I also squashed the commits, hope this is fine.